src/components/accordion/Accordion.jsx

```diff
@@ -43,34 +42,17 @@ const Accordion = ({
   className = '',
   spacing = 's',
 }: PropType) => {
-  const [opened, setOpened] = useState({});
-
-  const handleChange = useCallback(
-    (item: string, value: boolean) => {
-      if (allowMultiple) {
-        setOpened({...opened, [item]: value});
-      } else {
-        const newOpened = {...opened};
+  const [state, dispatch] = useReducer(accordionReducer, {
+    opened: {},
+    noGapBetweenElements: spacing === 'none',
+    allowMultiple,
```

Comment:
This will break when allowMultiple prop changes

Reply:
I think this is similar to `defaultOpened` - if we remove it from deps, it won't update if anyone decides that it should be dynamic. Here `allowMultiple` also should not be dynamic I guess (I can't find a use case when it should 🤔 )